code/game/objects/items/inducer.dm

```diff
@@ -247,22 +247,27 @@
 	powerdevice = /obj/item/stock_parts/power_store/battery/super
 
 /obj/item/inducer/cyborg
-	name = "internal inducer"
+	name = "modular inducer"
 	icon = 'icons/obj/tools.dmi'
 	icon_state = "inducer-engi"
-	powerdevice = null
-
-/obj/item/inducer/cyborg/add_context(atom/source, list/context, obj/item/held_item, mob/user)
-	return NONE
 
 /obj/item/inducer/cyborg/examine_hints(mob/living/user)
-	return list()
+	. = list()
 
-/obj/item/inducer/cyborg/get_cell(atom/movable/interface, mob/living/silicon/robot/silicon_friend)
-	return istype(silicon_friend) ? silicon_friend.cell : null
+	var/obj/item/stock_parts/power_store/our_cell = get_cell(src, user)
+	if(!QDELETED(our_cell))
+		. += span_notice("The cell can be removed with a crowbar.")
+		. += span_notice("Its display shows: [display_energy(our_cell.charge)].")
+		if(opened)
+			. += span_notice("Plasma sheets can be used to recharge the cell.")
+	else
+		. += span_warning("It's missing a power cell.")
+	. += span_notice("Its battery compartment can be [EXAMINE_HINT("screwed")] [opened ? "shut" : "open"].")
```

Comment:
Maybe I am missing something here but is this inducer not now generally identical to a normal one except that it cannot be used to charge cyborgs?  
You have a line in the examine string that implies that the cell can be removed with a crowbar but do not seem to have implemented that interaction as far as I can tell. Am I missing it, did you miss it, or is this wrong? Is there a reason you need an override to this proc instead of using the base version?

Reply:
![image](https://github.com/user-attachments/assets/ade7b044-7f9e-4ef7-a55c-73d39bd879b3)

Reply:
I had that originally just be a passthrough yeah